lxd/images.go

```diff
@@ -2472,6 +2473,7 @@ func imageSyncBetweenNodes(d *Daemon, project string, fingerprint string) error
 		return nil
 	}
 
+	// Get a liste of nodes that do not have the address.
```

Comment:
Typo

Reply:
Fixed, this is the only change I made to the branch so past test results should stand and shouldn't block this.